### PR TITLE
Used images that moved from docker hub to quay for db integration testing

### PR DESCRIPTION
**Pull request by skabashnyuk**
### What does this PR do?
Used images that moved from docker hub to quay for db integration testing

### Screenshot/screencast of this PR
n/a

### What issues does this PR fix or reference?
Partially https://github.com/eclipse/che/issues/18292

### How to test this PR?
- `mvn clean install - Pitegration`

### PR Checklist

[As the author of this Pull Request I made sure that:](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#pull-request-template-and-its-checklist)

- [x] [The Eclipse Contributor Agreement is valid](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-eclipse-contributor-agreement-is-valid)
- [x] [Code produced is complete](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-produced-is-complete)
- [x] [Code builds without errors](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-builds-without-errors)
- [x] [Tests are covering the bugfix](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#tests-are-covering-the-bugfix)
- [x] [The repository devfile is up to date and works](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-repository-devfile-is-up-to-date-and-works)
- [x] [Sections `What issues does this PR fix or reference` and `How to test this PR` completed](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#sections-what-issues-does-this-pr-fix-or-reference-and-how-to-test-this-pr-completed)
- [x] [Relevant user documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [x] [Relevant contributing documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [x] [CI/CD changes implemented, documented and communicated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#cicd-changes-implemented-documented-and-communicated)

### Reviewers

Reviewers, please comment how you tested the PR when approving it.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -65,6 +65,8 @@
         <commons-fileupload.version>1.3.3</commons-fileupload.version>
         <commons-io.version>2.6</commons-io.version>
         <commons-lang.version>2.6</commons-lang.version>
+        <integration.mysql.db.image>quay.io/eclipse/che--centos--mysql-57-centos7:latest-e08ee4d43b7356607685b69bde6335e27cf20c020f345b6c6c59400183882764</integration.mysql.db.image>
```
Comment (benoitf) [inline pom.xml:69]: it should use the one from sre mirror or from registry.centos.org/centos that are already mirrored

Comment (skabashnyuk) [reply pom.xml:69]: are you sure it's the same image?

Comment (benoitf) [reply pom.xml:69]: on https://registry.centos.org/centos/postgresql-96-centos7/latest (description tab) and on https://hub.docker.com/r/centos/postgresql-96-centos7 it seems that we see the same Dockerfile/source repository

image

Comment (l0rd) [reply pom.xml:69]: @skabashnyuk we don't want to maintain those images, hence I would rely on the ones maintained by app-sre.

Comment (nickboldt) [reply pom.xml:69]: See comments on https://github.com/eclipse/che/issues/18292#issuecomment-730412565

Comment (skabashnyuk) [reply pom.xml:69]: I think we can do that for mysql easily. For PostgreSQL, I would prefer to wait for @tolusha 's commits in operator and later will reuse the same.

**Comment by skabashnyuk** [comment]
I was not able to quickly use `quay.io/app-sre/mysql:5.7.7` for MySQL testing. However, tests of `registry.centos.org/centos/postgresql-96-centos7` were pretty successful.
I would like to create a separate issue for migrating to `quay.io/app-sre/*` images

**Review by tolusha — APPROVED** [review]

**Comment by nickboldt** [comment]
> I would like to create a separate issue for migrating to quay.io/app-sre/* images

Please do. 

Also are these `postgresql-96-centos7` images multi-arch? doesn't seem like it, but I'm gettting unexpected registry information when I use `skopeo inpect docker://... --raw` so the registry itself might be publishing different/incomplete metadata, thus I'm not sure how to get which arches are supported.

For the rhel8 based version, I get this:

```
$➔ skopeo inspect docker://registry.redhat.io/rhel8/postgresql-96:1 --raw | yq '.manifests[].platform.architecture' -r
amd64
arm64
ppc64le
s390x
```

**Comment by skabashnyuk** [comment]
@nickboldt multi-arch is not in the scope of this pr.

**Pull request merged**